ash/src/entry.rs

```diff
@@ -83,6 +83,11 @@ pub trait EntryV1_0 {
         allocation_callbacks: Option<&vk::AllocationCallbacks>,
     ) -> Result<Self::Instance, InstanceError>;
 
+    unsafe fn create_instance_from_raw_handle(
```

Comment:
I think I would rather see these helper functions implemented on the types directly, unless there is a specific reason? The traits mostly consist of methods with a default impl.

Reply:
I assumed it was a requirement since they rely on the `Instance` type defined in the trait.

Reply:
If it's implemented on the type directly, it only needs to return a single specific type, so that shouldn't be an issue.

Reply:
@Jasper-Bekkers are you interested in addressing this?